src/components/VideosModal.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import PropTypes from "prop-types";
+import {Collapsible, CollapsibleItem} from 'react-materialize';
+
+export default function VideosModal({ setVideoModal }) {
+  return (
+    <div id="modal" className="modal-container">
+      {/* React Materalize import */}
+      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css"></link>
```

Comment:
Huh. I [looked it up](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link) and it's actually valid to use a `<link rel="stylesheet">` within the `<body>` of an HTML file. Who knew?!

Good to know that it's not a well-accepted practice and that most (all?) other types of `<link>` tags are not allowed within the `<body>` tag.